### ISSUE TITLE
Validation set for CMNIST

### ISSUE THREAD

**Issue by googlebaba** [kakaoenterprise/Learning-Debiased-Disentangled#1]
Dear authors,
I want to know the data setting of validation set for CMNIST. Does the validation set has the same bias degree with training set? Thanks in advance.

**Comment by leebebeto** [comment]
Thanks for your interest in our work. Like you asked, the bias degree of the validation set is same as that of the training set.

**Comment by googlebaba** [comment]
Thanks for the reply. From the code, it seems that you directly pick the best test results without using the validation set, correct?

**Comment by leebebeto** [comment]
We used the same bias severity as used in the unbiased test set for tuning the hyper parameters. Thank you.

**Comment by googlebaba** [comment]
Thanks, so, I wonder what's the function of the validation set?

**Comment by leebebeto** [comment]
We used the validation set for evaluating the classification accuracy on bias-aligned samples mainly.
Thank you!
